client-src/elements/chromedash-admin-feature-links-page.ts

```diff
@@ -28,21 +47,26 @@ export class ChromedashAdminFeatureLinksPage extends LitElement {
       `,
     ];
   }
-  static get properties() {
-    return {
-      loading: {type: Boolean},
-      sampleId: {type: String},
-      samplesLoading: {type: Boolean},
-      featureLinksSamples: {type: Array},
-      featureLinksSummary: {type: Object},
-    };
-  }
 
-  constructor() {
-    super();
-    this.featureLinksSummary = {};
-    this.featureLinksSamples = [];
-  }
+  @property({type: String})
+  sampleId = '';
+  @property({type: Boolean})
+  samplesLoading = false;
+  @state()
+  loading = true;
+  @state()
+  featureLinksSamples: FeatureLink[] = [];
+  @state()
+  featureLinksSummary: FeatureLinksSummary = {
+    total_count: 0,
+    covered_count: 0,
+    uncovered_count: 0,
+    error_count: 0,
+    http_error_count: 0,
+    link_types: [],
+    uncovered_link_domains: [],
+    error_link_domains: [],
+  };
```

Comment:
I'd avoid the default where you have to invent field values, and just use

```suggestion
  featureLinksSummary!: FeatureLinksSummary;
```